android/sdk/src/main/cpp/app/organicmaps/sdk/Framework.cpp

```diff
@@ -1776,7 +1776,8 @@ JNIEXPORT jobject JNICALL Java_app_organicmaps_sdk_Framework_nativeGetProductsCo
   static jmethodID const productsConfigConstructor =
       jni::GetConstructorID(env, productsConfigClass, "(Ljava/lang/String;[Lapp/organicmaps/sdk/products/Product;)V");
 
-  jni::TScopedLocalRef const placePagePrompt(env, jni::ToJavaString(env, config->GetPlacePagePrompt()));
+  jni::TScopedLocalRef const placePagePrompt(
+      env, jni::ToJavaString(env, config->placePagePrompt ? *config->placePagePrompt : ""));
```

Comment:
Why not keep convenient selector functions (GetPlacePagePrompt) in a struct?

Reply:
What's the benefit? It's a simple struct with 1-to-1 mapping of fields to json file. Adding getters is an overhead here.